sale_team_default_values/README.rst

```diff
@@ -0,0 +1,4 @@
+Sales Teams Default Values
```

Comment:
Hi @lepistone thanks for your contribution.

Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst